dev-support/create-release/README.txt

```diff
@@ -105,6 +105,9 @@ $ scp ~/gpg.example.apache.pub example.gce.host:
 #   gpg-agent's extra socket (this will restrict what commands the remote node is allowed to have
 #   your agent handle. Note that the gpg guide above can help you set this up in your ssh config
 #   rather than typing it in ssh like this every time.
+# Note that as of maven-gpg-plugin, with gnupg >= 2.1, the plugin uses `--pinentry-mode error`,
```

Comment:
this should have been "as of maven-gpg-plugin 3.0.1"

Reply:
Yeah, lets see if we add a launch flag like Duo suggests. I'll update the readme accordingly.